File: Contas/client/src/app/shared/params/pagador.params.ts
import { Params } from "@/core/models/params.model";

/**
 * @author Marcelo M. de Castro
 * @summary Parametro de consulta do Pagador
 * @version 1.0
 */
export class PagadorParams extends Params {
    nome: string = '';
    cpf: number = 0;
    email: string = '';
}

File: Contas/server/Contas.Core/Specifications/Params/PagadorParams.cs
using Contas.Core.Specifications.Base;

namespace Contas.Core.Specifications.Params;

public class PagadorParams : SpecificationParams
{
    public string? Nome { get; set; }
    public string? Email { get; set; }
    public long? CPF { get; set; }
}
